src/frontend/src/views/Homepage/index.test.tsx

```diff
@@ -13,6 +13,7 @@ test("renders the Homepage", async () => {
   expect(screen.getByText("Loading...")).toBeInTheDocument();
 
   await waitFor(() => {
-    expect(screen.getByText(/Welcome to CZ Gen Epi/)).toBeInTheDocument();
+    expect(screen.getByText(/Welcome/i)).toBeInTheDocument();
+    expect(screen.getByText(/to CZ GEN EPI/i)).toBeInTheDocument();
```

Comment:
There's actually a <br /> line break between `Welcome` and the rest of the text. That's why it is necessary to have these two `expect` statements.